data/human/human missions.txt

```diff
@@ -7411,3 +7411,1824 @@ mission "Adelita 5"
 			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`
 			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
 			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`
+
+
+
+mission "Timothy Radrickson 1: Stowaway"
+	invisible
+	landing
+	source
+		near "Zeta Aquilae" 3 10
+		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"
+	destination "Rand"
+	to offer
+		random < 10
+	on enter
+
+	# You will see this conversation after the first jump you make.
+	on enter
+		conversation
+			`Upon completing your jump, one of your internal sensors begins to beep. You flick off the siren and check the diagnostic report: apparently there is a non-negligible heat source detected in the food storage of your ship.`
+			`	As problems go, it is not a particularly alarming one: there are plenty of reasons there may be a heat spike in storage, and most of them would not require the captain of the ship to investigate.`
+			`	It is unusual, however.`
+			choice
+				`	(Investigate.)`
+					goto check
+				`	(Ignore the sensor.)`
+
+			action
+				fail
+				event "timothy radrickson: mummy" 41
+			`	It's almost certainly nothing, and you have more important things to do than chase down every little sensor glitch on your ship.`
+
+			label check
+			`	After a short search, and the rearranging of quite a few deceptively heavy boxes, you are able to track the heat signature to a foodstuff crate you recently took on. It has yet to be opened, but you can already tell its seal has been tampered with.`
+			choice
+				`	(Open the crate.)`
+					goto crate
+				`	(Dump the crate out the airlock.)`
+
+			action
+				fail
+			`	As the crate tumbles out into lifeless space your onboard sensors quickly note the anomalous heat signature disappear. Whatever was in that crate is no longer your problem, and probably won't be anyone else's either.`
```

Comment:
There’s no ending here, so will that mean it goes to `label crate`?

Reply:
i added a decline to after that. Though I am not sure what bene was doing with the action fail stuff so hopefully I didn't mess that up.

Reply:
I think the action fail part only happens after a day passes. `decline` the way you put it in should work immediately.